modules/tiled-gallery.php

```diff
@@ -30,3 +30,21 @@ function jetpack_tiled_gallery_configuration_load() {
 }
 
 jetpack_load_tiled_gallery();
+
+// Tile-gallery block definition can be found in wp-calypso repo
+jetpack_register_block( 'tiled-gallery', array(
+	'render_callback' => 'jetpack_tiled_gallery_load_assets'
+) );
+
+/**
+ * Renders the tiled gallery dynamically to the user
+ *
+ * @param $attr array - array of attributes
+ * @param $content string - content block
+ *
+ * @return mixed
```

Comment:
It appears that we're returning the original `$content`, and nothing is modifying it, so I'd expect this to still be a string.